tensorflow/core/platform/env.h

```diff
@@ -205,12 +205,14 @@ class Env {
   Status RenameFile(const string& src, const string& target);
 
   // TODO(jeff,sanjay): Add back thread/thread-pool support if needed.
-  // TODO(jeff,sanjay): if needed, tighten spec so relative to epoch, or
-  // provide a routine to get the absolute time.
+
+  /// \brief Returns the current time measured in the number of microseconds
+  /// since the Unix Epoch.
+  virtual uint64 TimeMicros() = 0;
 
   /// \brief Returns the number of micro-seconds since some fixed point in
   /// time. Only useful for computing deltas of time.
-  virtual uint64 NowMicros() = 0;
+  virtual uint64 NowMicros() { return TimeMicros(); }
```

Comment:
Let's just tighten the guarantees of the existing functions rather than create a new one? It seems all instances of it use the same epoch already anyway.